core/spec.md

```diff
@@ -3846,7 +3965,7 @@ and the following Resource level attributes:
   - When not present, the default value is `false`.
   - REQUIRED when `true`, otherwise OPTIONAL.
   - If present, it MUST be a case sensitive `true` or `false`.
-  - If present in a request, a value of `null` has the same meaning as
+  - If present in a request, a value of `null` MUST have the same meaning as
```

Comment:
Do we have to support null as a valid value at all? Do we have the same null==false equivalence for other Boolean attributes?